packages/menu-bar/theme/material/vaadin-menu-bar-button-styles.js

```diff
@@ -14,8 +14,8 @@ const menuBarButton = css`
     justify-content: center;
   }
 
-  :host([theme='outlined']),
-  :host([theme='contained']) {
+  :host([theme~='outlined']),
```

Comment:
I had to replace `=` with `~=` as the new variant gets propagated to buttons so they end up `theme="outlined end-aligned"`. Added visual tests to ensure this works correctly in Material theme.

Looks like this isn't needed in case of Lumo where selectors are already correct.